package.json

```diff
@@ -76,7 +76,7 @@
         "setup": "yarn install --frozen-lockfile; npm-run-all clean build:i18n",
         "start": "npm-run-all setup start:examples",
         "start:dev": "yarn setup; LANGUAGE=en-US BABEL_ENV=development NODE_ENV=development node --max_old_space_size=8192 node_modules/webpack-dev-server/bin/webpack-dev-server.js --config scripts/webpack.config.js --mode development",
-        "start:examples": "EXAMPLES=true LANGUAGE=en-US REACT=true BABEL_ENV=development NODE_ENV=development node --max_old_space_size=8192 node_modules/react-styleguidist/bin/styleguidist.js server --config scripts/styleguide.config.js --mode development",
+        "start:examples": "EXAMPLES=true LANGUAGE=en-US REACT=true BABEL_ENV=development NODE_ENV=development node --max_old_space_size=8192 node_modules/react-styleguidist/lib/bin/styleguidist.js server --config scripts/styleguide.config.js --mode development",
```

Comment:
we can use `node_modules/.bin/styleguidist` which is less likely to have changes in the future. can  you also update `build:prod:examples`?

Reply:
it throw errors from the `bin` since it wasnt located in the `bin` but in `lib/bin`

Reply:
the `.bin` folder in the root level node_modules should be able to run the command too right?